x-pack/plugins/observability_solution/infra/public/components/logging/log_statusbar.tsx

```diff
@@ -6,20 +6,17 @@
  */
 
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import styled from '@emotion/styled';
 
-import { euiStyled } from '@kbn/kibana-react-plugin/common';
-
-export const LogStatusbar = euiStyled(EuiFlexGroup).attrs(() => ({
-  alignItems: 'center',
-  gutterSize: 'none',
-  justifyContent: 'flexEnd',
-}))`
-  padding: ${(props) => props.theme.eui.euiSizeS};
-  border-top: ${(props) => props.theme.eui.euiBorderThin};
+export const LogStatusbar = styled(EuiFlexGroup)`
+  padding: ${(props) => props.theme.euiTheme.size.s};
+  border-top: ${(props) => props.theme.euiTheme.border.thin};
   max-height: 48px;
   min-height: 48px;
-  background-color: ${(props) => props.theme.eui.euiColorEmptyShade};
+  background-color: ${(props) => props.theme.euiTheme.colors.emptyShade};
   flex-direction: row;
 `;
 
+LogStatusbar.defaultProps = { alignItems: 'center', gutterSize: 'none', justifyContent: 'flexEnd' };
```

Comment:
**nit**: this is a deprecated React feature, prefer switching over a function react component if you need default params.

Reply:
Found this as a [suggestion to replace the attrs on styled components](https://github.com/emotion-js/emotion/issues/821#issuecomment-415697393) but you're right it's a bit old, I'll see if I found something else

Reply:
Hope this helps, it should make the trick for this case (coded only, check if  everything works fine on Kibana please):
```tsx
export const LogStatusbar = (props: EuiFlexGroupProps) => {
  const { euiTheme } = useEuiTheme();

  return (
    <EuiFlexGroup
      alignItems="center"
      gutterSize="none"
      justifyContent="flexEnd"
      css={css`
        padding: ${euiTheme.size.s};
        border-top: ${euiTheme.border.thin};
        max-height: 48px;
        min-height: 48px;
        background-color: ${euiTheme.colors.emptyShade};
        flex-direction: row;
      `}
      {...props}
    />
  );
};
```